helpers.js

```diff
@@ -99,6 +110,13 @@ let getFiles = (path) => {
     return glob.sync("**/*.js", {'ignore': ['node_modules/**/*']});
 };
 
+/* Get all js files for test
+ * Return path of all js files
+ */
+let getTestFiles = (path) => {
+    return glob.sync("**/*.spec.js", {'ignore': ['node_modules/**/*']});
```

Comment:
Bumped into this poll today :smile:

https://twitter.com/dan_abramov/status/762662306618732544